src/ServerScriptService/MainSystem.server.lua

```diff
@@ -4,23 +4,21 @@ local Players = game:GetService("Players")
 -- Example #1: Kicking Players if Datastores are down.
 
 Players.PlayerAdded:Connect(function(player)
-    if DowntimeService:GetDatastoreAPIStatus() == "down" then
-        player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
-    end 
+	if DowntimeService.statusFromAlias("datastore")== "Down" then
+		player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
```

Comment:
It's likely best that kicking is a toggleable feature rather than always kicking.